### DISCUSSION TITLE
How to access "audit" logs? (reports optionnaly created for security-relevant events)

### DISCUSSION THREAD

**Discussion by jvagnet** [photoprism/photoprism#3050] [Q&A]
Hi everyone,

I'd like to be able to access [this log](https://github.com/photoprism/photoprism/blob/cc38922cbe2611ddd87142e90e12300cd29ed0a5/internal/api/auth_session_create.go#L24).
If my understanding of the code is correct, this should be logged whenever an authentication api call fails. (I'd like to implement a custom fail2ban rule, and I would need to access this log in order to retrieve the IP address of the failing call). But the only log I can see on a failed authentication is: `photoprism-photoprism-1  | time="2023-01-02T15:24:55Z" level=debug msg="server: POST /api/v1/session (401) [903.193146ms]"`

My instance is deployed with docker, and I have no issue with accessing the logs, but it looks like there is something different with these "Audit" logs. As stated [here](https://github.com/photoprism/photoprism/blob/cc38922cbe2611ddd87142e90e12300cd29ed0a5/internal/event/audit.go#L20): "_Audit optionally reports security-relevant events_", but I can't figure out how to enable this option. I have tried changing the log level in the env config and enabling the "debug" mode but none of these seem to enable this log.

Are these audit logs sent to another output than the standard one? Or is there an option that I should set?

Thank you.

**Comment by lastzero** [comment]
That's something we're working on right now. Feel free to contact us at hello@photoprism.app if you would like to help us test it.
